packages/babel-preset/__fixtures__/duplicated-imports/code.ts

```diff
@@ -0,0 +1,12 @@
+import { dupImport1 } from 'custom-package';
+import { createStyles } from 'custom-package';
+import { dupImport2 } from 'another-custom-package';
+import { otherImport } from 'another-package';
+
+dupImport1;
+dupImport2;
+otherImport;
```

Comment:
Can you please turn these into real cals to be sure that transforms will be applied? 🐱 

```suggestion
export const useClassesA = createStylesA({
  root: { color: 'yellow' },
})
export const useClassesB = createStylesB({
  root: { color: 'blue' },
})
export const useClassesC = createStylesC({
  root: { color: 'orange' },
})
```

Reply:
Addressed as suggested. Thanks!